test/TestProjects/FirstTest-Cadl/Generated/ExtensibleEnum.Serialization.cs

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Extensible enum should not have serialization class. See: https://github.com/Azure/autorest.csharp/issues/2528

Also check the test case: https://github.com/Azure/autorest.csharp/blob/79ca039d08177cc16672dace1161c7fd68ec9e9b/test/AutoRest.TestServerLowLevel.Tests/LowLevel/Generation/EnumWritingTests.cs#L11-L15

Reply:
It is extensible enum

Reply:
@archerzz, as Extensible enum should not have serialization class, why this class get auto generated?

Reply:
resolved